OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -58,9 +63,58 @@ public void PlayMovieFullscreen(string movie, LuaFunction func = null)
 				};
 			}
 			else
-				onComplete = () => { };
+				onCompleteFullscreen = () => { };
 
-			Media.PlayFMVFullscreen(world, movie, onComplete);
+			Media.PlayFMVFullscreen(world, movie, onCompleteFullscreen);
+		}
+
+		Action onLoadComplete;
+		Action onCompleteRadar;
+		[Desc("Play a VQA video in the radar window.  File name has to include the file extension.")]
+		public void PlayMovieInRadar(string movie, LuaFunction func = null)
+		{
+			if (func != null)
+			{
+				var f = func.CopyReference() as LuaFunction;
+				onCompleteRadar = () =>
+				{
+					try
+					{
+						using (f)
+							f.Call();
+					}
+					catch (LuaException e)
+					{
+						Context.FatalError(e.Message);
+					}
+				};
+			}
+			else
+				onCompleteRadar = () => { };
+
+			Stream s = null;
+			try
+			{
+				s = GlobalFileSystem.Open(movie);
+			}
+			catch (FileNotFoundException e)
+			{
+				Context.FatalError(e.Message);
```

Comment:
I think this is not a good idea. We cannot rely on people having the movies installed.